Gemfile

```diff
@@ -13,6 +13,6 @@ gem "rb_sys"
 gem "rspec", "~> 3.0"
 
 gem "standard", "~> 1.3"
-gem 'pry', '~> 0.14.2'
+gem "pry", "~> 0.14.2"
```

Comment:
I should probably just remove this, I think it's not actually being used and was just there for some debugging